<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>About</class>
 <widget class="QDialog" name="About">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GitUI About</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelProgramName">
     <property name="text">
      <string>programName</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelVersion">
     <property name="text">
      <string>version</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelAuthor">
     <property name="text">
      <string>Author</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelEmail">
     <property name="text">
      <string>e-mail</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonBack">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
